tsconfig.json

```diff
@@ -11,14 +11,13 @@
         "allowUnreachableCode": false,
         "allowUnusedLabels": false,
         "esModuleInterop": true,
-        "isolatedModules": true,
         "jsx": "react-native",
         "types": ["react-native", "node", "jest"],
         "moduleResolution": "node",
         "noEmit": false,
         "strict": true,
         "importHelpers": true,
-        "importsNotUsedAsValues": "error",
+        "verbatimModuleSyntax": true,
```

Comment:
This isn't directly related to this PR, but why are we using `importsNotUsedAsValues`/`verbatimModuleSyntax` in the mobile app? I'm just curious since that isn't something we've turned on on the web side, and I've spent a lot of time recently trying to understand what everything in the tsconfig does.

I do assume this is the reason that you've needed to be explicit about when types are being imported now though

Reply:
is definitely to keep the code clean, specially when it gets bundled as types are not really required in a bundle.

as of this version of TS, `importsNotUsedAsValues` is deprecated in favor of `verbatimModuleSyntax`

Reply:
I think the default behaviour is to strip out any imports that are only used as types though. It sounded to me like the only time you'd only need to change that if you were importing types from a file that had side effects. Maybe that's not the case though